### PR TITLE
Refactor Brand Rule

### PR DESCRIPTION
**Pull request by zsteiner**
For easier implementation on Hybris side. 

Previously brand-rule was a <hr>. Now brand rule can be implemented simply by applying a class. Added color variants (grey and white).

### CODE REVIEW

File: src/materials/atoms/brand-rule.html [hunk 1]
```diff
@@ -1 +1,3 @@
-<hr class="hpe-brand-rule">
\ No newline at end of file
+<h1 class="hpe-headline hpe-headline--black hpe-headline--caps hpe-brand-rule">
+  This is sample headline with a brand rule
+</h1>
```
Comment (ashblue) [inline src/materials/atoms/brand-rule.html]: Don't forget to add a blank line to the end of files in order to drastically reduce the possibility of merge conflicts.

File: src/materials/organisms/carousel.html [hunk 2]
```diff
@@ -3,8 +3,7 @@
   <section class="hpe-carousel__image {{this.class}}" style="background-image: url('{{this.imageURL}}')">
     <div class="hpe-carousel__content hpe-centered-section">
       <div class="hpe-carousel__card">
-        <h1 class="hpe-headline hpe-headline--medium hpe-headline--caps hpe-headline--black hpe-carousel__title hpe-page-headline">{{this.title}}</h1>
-        {{> brand-rule }}
+        <h1 class="hpe-headline hpe-headline--medium hpe-headline--caps hpe-headline--black hpe-brand-rule hpe-carousel__title hpe-page-headline">{{this.title}}</h1>
```
Comment (ashblue) [inline src/materials/organisms/carousel.html]: Line is getting a bit long. Might be worth breaking the element inner HTML onto another line.

**Review by ashblue — APPROVED** [review]
Minor comments. Good to merge

File: src/materials/atoms/brand-rule.html (hunk 1 above)
Comment (zsteiner) [reply src/materials/atoms/brand-rule.html]: Done!

**Pull request merged**